src/components/PeopleFilters.tsx

```diff
@@ -27,66 +74,52 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
+            {FILTER_CENTURY.map((century: string) => {
+              const isCenturyIncluded = centuries.includes(century);
 
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+              return (
+                <SearchLink
+                  key={century}
+                  data-cy="century"
+                  className={classNames(
+                    'button',
+                    'mr-1',
+                    { 'is-info': isCenturyIncluded },
+                  )}
+                  params={{
+                    centuries: isCenturyIncluded
+                      ? centuries.filter(cent => cent !== century)
+                      : [...centuries, century],
```

Comment:
Create variable for this condition, the same as you did it for `isCenturyIncluded`